admin_guide/allocating_node_resources.adoc

```diff
@@ -11,76 +11,69 @@
 
 toc::[]
 
-== Overview
+== Purpose for allocating node resources
 
 To provide more reliable scheduling and minimize node resource overcommitment,
-each node can reserve a portion of its resources for use by all underlying
-xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[node
-components] (e.g., kubelet, kube-proxy, Docker) and the remaining system
-components (e.g., *sshd*, *NetworkManager*) on the host. Once specified, the
-scheduler has more information about the resources (e.g., memory, CPU) a node
-has allocated for pods.
+you can reserve a portion of the CPU and memory resources for use by the underlying xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[node components] such as kubelet, kube-proxy, and the container engine. The resources that you
+reserve are also used by the remaining system components such as *sshd*, *NetworkManager*, and so on. After you specify the resources to reserve, the scheduler has more information about the remaining memory and CPU resources that a node has available for use by pods.
 
 [[allocating-node-settings]]
-== Configuring Nodes for Allocated Resources
+== Configuring nodes for allocated resources
 
-Resources reserved for node components are based on two node settings: `kube-reserved` and `system-reserved`.
+Resources are reserved for node components and system components in {product-title} with the `system-reserved` node setting.
 
-You can use the `kube-reserved` setting to reserve a portion of memory dedicated for kubelet and the `system-reserved` setting for the rest of the system processes. 
+{product-title} does not use the `kube-reserved` setting. Documentation for Kubernetes and some cloud vendors that provide a Kubernetes environment might suggest configuring `kube-reserved`. That information does not apply to a {product-title} cluster.
 
-[NOTE]
-====
-Use caution when setting these parameters. Enforcing `kube-reserved` and `system-reserved` limits
-can lead to critical system services on that node being starved of CPU or OOM killed. For more information
-on the effects of these settings, see xref:computing-allocated-resources[Computing Allocated Resources].
-====
+Use caution when you tune your cluster with resource limits and enforcing limits with evictions. Enforcing `system-reserved` limits can lead to starving critical system services from CPU or killing the critical system services by the OOM killer.
```

Comment:
s/enforcing/enforce
Is there a better/more technical term than "starving" here? We should avoid anthropomorphism. 

Possibly the same for "killing", unless it seems necessary to refer to the OOM killer in this example. Perhaps, "...or ending the critical system services when critical memory is running low."